### DISCUSSION TITLE
getting unknown system error in node fs on EFS mounted directory

### DISCUSSION THREAD

**Discussion by aman-gupta-doc** [nodejs/help#5045] [Q&A]
Hi everyone,
I got below errors multiple times when using node fs write or read functions.
```
Error: Unknown system error -116: Unknown system error -116
```

Why I am getting those errors?

**Comment by bnoordhuis** [comment]
Linux, right? EFS is a network file system and 116 is EISCONN, so presumably the connection with the backend got in some kind of bad state.

I doubt node is the culprit here, it's probably just the messenger. That error code comes straight from the kernel. Check the kernel logs.

**Reply by aman-gupta-doc** [reply]
Thank you, @bnoordhuis, for your answer.

Yes, My OS is Linux. As I can check in Linux error code list, error number `116` pointed to `ESTALE` and for `EISCONN` they have error number `106`. 
Can you help me with understanding why I got `ESTALE` on writing file?

Error code list
https://www.thegeekstuff.com/2010/10/linux-error-codes/
